python/tests/test_scip.py

```diff
@@ -42,3 +42,122 @@ def test_set_param(model, name, param_type):
         value = param_type(1)  # Cast one to the required type
     model.set_param(name, value)
     assert model.get_param(name) == value
+
+
+@pytest.mark.parametrize("name,param_type", names_types)
+def test_get_set_param(model, name, param_type):
+    value = model.get_param(name)
+    model.set_param(name, value)
+    assert model.get_param(name) == value
```

Comment:
Note that if `set_param` is broken and does not do anything, this test will always succeed.
I would be nice to first set some value, then compare with get, perhaps in the last test?

Reply:
Well there exist no test that does try all potential bugs at once. This one checks that get(set()) is a no-op. Another test is already in place, that checks that set(some_value) is not a no-op. So I believe we're ok.